src/main/java/com/theelixrlabs/healthcare/model/DoctorsEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.theelixrlabs.healthcare.model;
+
+import com.theelixrlabs.healthcare.constants.DoctorConstants;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.UUID;
+
+/**
+ * Entity class representing doctors in the healthcare system.
+ * This class is mapped to the MongoDB collection specified by DoctorConstants.DOCTOR_COLLECTION_NAME.
+ */
+@Data
+@Document(collection = DoctorConstants.DOCTOR_COLLECTION_NAME)
+public class DoctorsEntity {
+    @Id
+    private UUID id;
+    private String firstName;
```

Comment:
Are these fields nullable? It is a good practice to constraint validations to these fields.

Reply:
Done addeed the validations for the fields.